### ISSUE TITLE
release?

### ISSUE THREAD

**Issue by chiamtc** [weikee94/node-todos-api#1]
when are you releasing this app so I can track my notes?

**Comment by weikee94** [comment]
@chiamtc plan to using mlab as database instead of firebase (will add JWT token soon for user login)

**Comment by chiamtc** [comment]
this is a course? mind if I have the link of it? 💪

**Comment by weikee94** [comment]
@chiamtc yup this is course from udemy, here you go [https://www.udemy.com/the-complete-nodejs-developer-course-2/learn/v4/t/lecture/5795044?start=510](url)

**Comment by chiamtc** [comment]
OHHH, I also bought it but never started it.
